### DISCUSSION TITLE
Way of padding whole numbered list

### DISCUSSION THREAD

**Discussion by Serenade600** [marp-team/marp#117] [Q&A] [answered]
Hi all,

As I want to pad a numbered list a distance to left border , I added 
```
padding-left: 8%
```
into
```
 <style scoped>
 li { font-size: 40px;font-weight:bold}
 </style>
```
,which finally became
```
<style scoped>
li { padding-left: 8%;font-size: 40px;font-weight:bold}
</style>
```
.

However, it seems that only the text got padded. And their numbers stay where they are. So my question is, is there a way to padding the numbers along with the text?

Thanks in advance!

**Comment by yhatt** [comment, marked as answer]
The default padding of list is defined to `<ol>` element by the browser. It is not `<li>` element. It applies in general CSS styling, not only Marp.
https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_lists

So set `padding-left` to `ol` to tweak padding of the list.

```
ol {
  /* Example: no padding */
  padding-left: 0;
}
```

https://jsfiddle.net/qcLud6k2/2/
